src/sprout/Helpers/ColModifierJsonArray.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/*
+ * Copyright (C) 2023 Karmabunny Pty Ltd.
+ *
+ * This file is a part of SproutCMS.
+ *
+ * SproutCMS is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU General Public License as published by the Free Software Foundation, either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * For more information, visit <http://getsproutcms.com>.
+ */
+
+namespace Sprout\Helpers;
+
+
+/**
+ * Helper for rendering out array data from a json stored string
+ *
+ * Does not HTML encode if preformatting.
+ *
+ * @package Sprout\Helpers
+ */
+class ColModifierJsonArray extends UnescapedColModifier
+{
+    private $_preformat;
+
+    /**
+     * @param bool $preformat Whether to wrap the content in a <pre> tag. Will escape otherwise
+     */
+    public function __construct(bool $preformat = true)
+    {
+        $this->_preformat = $preformat;
+    }
+
+    /**
+    * Modify a column value
+    * This value will be html/csv/etc encoded afterwards.
+    *
+    * @param mixed $val The incoming value
+    * @param string $field_name The name of the field being modified
+    * @return string The modified value
+    **/
+    public function modify($val, $field_name, $row)
+    {
+        $json_array = [];
+
+        if (!is_array($val)) {
+            // See if json..
+            $json_array = json_decode($val, true);
+            $json_error = json_last_error();
+
+            // If not json, just return the value. No exploding plz.
+            if ($json_error !== JSON_ERROR_NONE or !is_array($json_array)) {
+                return Enc::html($val);
+            }
+        }
+
+        $out = '';
+        foreach ($json_array as $key => $data) {
+            $key = ucwords(str_replace('_', ' ', $key));
```

Comment:
I suppose this assumes that the body is snake_case. Would we want to also prettify a JSON body that uses camelCase?

I think `Infector::humanize()` would sort it out. But you'd still need the ucwords(). I wonder if we need an `Inflector::title()` helper for that.

Reply:
I like that title helper idea, I actually went looking for it in Inflector and Text first...

Reply:
There are camelcase helpers in Text already, just not to sentence case

Reply:
I did some stuffs like you suggested. It feels nice.

It's also very late though, and I'm quite tired, so please give it a good eyeballing :)